### PR TITLE
update cfp with new closing date

### PR DESCRIPTION
**Pull request by ei8fdb**
extending cfp closing date.

### CODE REVIEW

File: _posts/2019-02-02-opensourcedesign-fosdem.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ By submitting a proposal you consent to be recorded and agree to license the con
 
 #### Submission deadline:
 
-1st December 2018
+8th December 2018
```
Comment (evalica) [inline _posts/2019-02-02-opensourcedesign-fosdem.md]: You could have made it like: 
--1st December 2018-- (Extended: 8th December 2018)

**Review by evalica — DISMISSED** [review]

File: _posts/2019-02-02-opensourcedesign-fosdem.md (hunk 1 above)
Comment (ei8fdb) [reply _posts/2019-02-02-opensourcedesign-fosdem.md]: good point. i'll update it now.

👍

Comment (ei8fdb) [reply _posts/2019-02-02-opensourcedesign-fosdem.md]: used this GH syntax: https://webapps.stackexchange.com/questions/14986/strikethrough-with-github-markdown

**Review by jancborchardt — APPROVED** [review]
Seems good!

**Review by evalica — APPROVED** [review]

**Pull request merged**